lib/players/human.rb

```diff
@@ -0,0 +1,9 @@
+class Human < Player # inherits from Player
+
+  def move(board)
+    # asks the user for input and returns it
+    puts "Please enter a move from 1 to 9"
+    input = gets.strip
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Please enter a move from 1 to 9"
  gets.strip
end